snap/snapcraft.yaml

```diff
@@ -4,8 +4,124 @@ summary: OpenStack Exporter for Prometheus
 license: MIT
 contact: solutions-engineering@lists.canonical.com
 description: |
-  The OpenStack exporter, exports Prometheus metrics from a running OpenStack
-  cloud for consumption by prometheus.
+  This snap uses openstack-exporter to provide OpenStack cloud metrics via Prometheus.
+  Grafana dashboards can then be used to visualize the exported metrics, see for example:\
+  https://grafana.com/grafana/dashboards/9701-openstack-dashboard/
+
+  **How-To**
+  ---
+
+  **How to install the snap:**
+
+     sudo snap install charmed-openstack-exporter
+
+     # To allow the snap to read from the /etc/openstack/, you may need to manually add the connection:
```

Comment:
nit: should this be a `must`? Or are you perhaps trying to convey that not everyone will need to read from `/etc/openstack`?

Reply:
I was referencing the upstream docs for that, maybe there is a point being made that there can be an autoconnect